src/llaminator.scss

```diff
@@ -49,3 +49,7 @@ llama-select-fab {
   bottom: 32px;
   right: 32px;
 }
+
+.mdc-card__media {
```

Comment:
would this make more sense in `llama-item.ts` directly?

Reply:
No, because we use open shadow DOM, i.e. global styles will be applied.